src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/ConfigurationBinderTests.Generator.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Microsoft.Extensions.SourceGeneration.Configuration.Binder.Tests
+{
+    public partial class ConfigurationBinderTests : ConfigurationBinderTestsBase
+    {
+        [Fact]
+        public void GeneratorHandlesInvocationsOnNewline()
+        {
+            IConfiguration configuration = TestHelpers.GetConfigurationFromJsonString(@"{""Longitude"":1,""Latitude"":2}");
+
+            GeolocationRecord record = configuration.Get<GeolocationRecord>();
+            Verify();
+
+            record = (GeolocationRecord)configuration
+                .Get(typeof(GeolocationRecord), _ => { });
```

Comment:
Should we be testing other permutations of line breaks that are permitted by the language to make sure we handle all of them?

Reply:
+1 

```C#
record = (GeolocationRecord)configuration.
                Get(typeof(GeolocationRecord), _ => { });

record = (GeolocationRecord)
                configuration.
                Get(typeof(GeolocationRecord), _ => { });

record = (GeolocationRecord)
                configuration
                .Get(typeof(GeolocationRecord), _ => { });
```